README.md

```diff
@@ -8,50 +8,67 @@ You must install the Emacs plugin (this) and a language server
 
 ### lsp-scala
 
-Install the `lsp-mode` and `sbt-mode` dependencies using your preferred package
-manager. Then clone this repo and load it:
+Clone this repo, and fetch `lsp-mode`, `lsp-ui`, and `sbt-mode` from
+MELPA with the package manager of your choice.  (We are working on
+adding this package to MELPA to make `lsp-scala` load like the rest.)
+Here is an example using `use-package`:
```

Comment:
The README is redundant now that it's documented in metals.  The advantage to us publishing instructions is that we can change them atomically with changes to the package, but the Metals instructions are better.  We should probably just link to those, and focus on keeping those current?

Reply:
The benefit of keeping the instructions on the Metals website is that the version numbers stay up-to-date with new releases. Also, the editor-agnostic parts of the text is shared with other editors so they stay in-sync as the instructions improve.

Reply:
Yes, I expect metals to evolve much faster than this few lines of LISP.  Once this is working solidly and in MELPA, let's get the Metals Emacs docs cleaned up and refer to them from here.